### PR TITLE
Define guidelines for labels management

### PR DESCRIPTION
**Pull request by damienfraud**
## Goals :soccer:
Define GitHub guidelines in terms of labels use and management.

## Inspiration :bulb:
- Sane GitHub Labels, on [Medium](https://medium.com/@dave_lunny/sane-github-labels-c5d2e6004b63)
- GitHub Labels and Milestones, on [SALTSTACK](https://docs.saltstack.com/en/2017.7/topics/development/labels.html)
- RayWenderlich Swift [Styleguide](https://github.com/raywenderlich/swift-style-guide/blob/master/README.markdown)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,68 @@ Templates should promote a high contribution quality by referring [contributing
 
 ## Label
 
+We believe GitHub labels should define immutable informations about issues.
```
Comment (romsi) [inline README.md]: We should not use `we` or `you` to describe the guideline.

File: README.md [hunk 2]
```diff
@@ -97,6 +97,68 @@ Templates should promote a high contribution quality by referring [contributing
 
 ## Label
 
+We believe GitHub labels should define immutable informations about issues. 
+
+We consider having to change labels over time to be a bad practice. Indeed, changing labels manually multiple times on issues can be very time consumming and prone to errors. Moreover, we believe that if the information about the issue is changing, it is surely not a relevant information to have as a label. Almost everytime, you will see that variable informations about an issue can be managed in another way which will be much more efficient in the end. If you consider this, you will tremendously reduce useless labels creation, and see that you have more than enough information through it.
```
Comment (romsi) [inline README.md]: We should not use `we` or `you` to describe the guideline.

File: README.md [hunk 3]
```diff
@@ -97,6 +97,68 @@ Templates should promote a high contribution quality by referring [contributing
 
 ## Label
 
+We believe GitHub labels should define immutable informations about issues. 
+
+We consider having to change labels over time to be a bad practice. Indeed, changing labels manually multiple times on issues can be very time consumming and prone to errors. Moreover, we believe that if the information about the issue is changing, it is surely not a relevant information to have as a label. Almost everytime, you will see that variable informations about an issue can be managed in another way which will be much more efficient in the end. If you consider this, you will tremendously reduce useless labels creation, and see that you have more than enough information through it. 
+
+We believe that labels should be regrouped into categories and that you only need three of them. We also recommend to use similar color styling accross categories for a stronger visual identification.
```
Comment (romsi) [inline README.md]: We should not use `we` or `you` to describe the guideline.

**Comment by damienfraud** [comment]
@romsi I added `Immutabilty` and `Colors` headlines, using *Why* pattern. I also removed `we` and `you` from guidelines.

**Comment by romsi** [comment]
@damienfraud I updated the milestone section. We should find a pattern that is visual in order to quickly get the important information. Text is good, but visual should be our guideline.

**Comment by damienfraud** [comment]
@romsi I updated the guidelines with a new formatting suggestion. 
I added the Why section again to make descriptions more concise. 
Also, I found the Preferred/Not Preferred format not optimal in this case. I suggest a Format with expendables examples instead, which still gives quick visual idea of the section without overloading the page with too many examples.
Finally, I really liked the use you have for Pro Tip section. 👍

**Comment by romsi** [comment]
@damienfraud we agreed on a new format. You need to fulfil the resources section and then we can merge it.

**Comment by romsi** [comment]
@damienfraud Please check my 2 latest commit. We should use the full title for links instead of using the source only.

**Preferred**

**Comment by romsi** [comment]
@damienfraud please check my latest commits. We should use the full title for links instead of the source name at the end.

** Preferred **

```markdown
- [A better collaboration using Github](https://github.com/blog/a-better-collaboration).
```

** Not Preferred **

```markdown
- A better collaboration using Github. [Github](https://github.com/blog/a-better-collaboration).
```

Imagine you are blind and you only get the information inside the brackets. Try to figure out what is the context and where are you going to end up using this link.

**Comment by damienfraud** [comment]
@romsi I'm ok with your changes. I will merge the PR.

**Pull request merged**